nimare/diagnostics.py

```diff
@@ -204,8 +202,14 @@ def transform(self, result):
 
             contribution_tables.append(contribution_table.reset_index())
 
-        # Concat PositiveTail and NegativeTail tables
-        contribution_table = pd.concat(contribution_tables, ignore_index=True, sort=False)
+        if pairwaise_estimators or len(label_maps) == 1:
```

Comment:
For MKDAChi2, the z-values can be positive or negative (you can check the z-map results from running the test in this pull request: https://github.com/neurostuff/NiMARE/pull/811).

I think we would want to display both Positive and Negative Tail cluster with Pairwise estimators.

Reply:
Also in the report, we only show the summary for `id1` and `coordinates1` from dataset1. Should we include another section for the second group?

Reply:
This is a little more questionable, overall, I would say yes, we do want to display `id2`  for the coordinates. for the contribution table, this is a bit debatable. With ALESubtraction, yes, I think it is useful to know how both groups contributed to the existing clusters.

For the MKDAChi2 I think it is less useful to see a contribution table (and with jacknife it would be computationally prohibitive if one was using the entire neurosynth dataset).

But the notion that ALESubtraction is used for more similarly sized groups and MKDAChi2 is used for comparing a relatively smaller dataset to a huge dataset is only reflective of how we are currently the algorithms, there's nothing stopping you from using ALESubtraction with neurosynth.

So I think the answer would be creating an option/flag for the user to decide whether they want to show the second group or not. could be called `display_second_group` or something, with a default value of `False`.